lib/hexpm_web/views/api/short_url_view.ex

```diff
@@ -6,6 +6,6 @@ defmodule HexpmWeb.API.ShortURLView do
   end
 
   def render("show", %{short_url: short_url}) do
-    %{short_code: short_url.short_code}
+    %{short_url: short_url}
```

Comment:
this was pretty confusing at first, I though `short_url = %ShortURL{}`. WDYT about changing param and variable name to simply `url`?

wdyt about also asserting on the response body in the corresponding controller test?

Reply:
good idea.